lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -8,7 +8,9 @@ def compound_changelog
       sort_tags_by_date(@filtered_tags)
       fetch_issues_and_pr
 
-      log = "#{@options[:header]}\n\n"
+      log = ''
+      log += @options[:frontmatter] if @options[:frontmatter]
```

Comment:
PStore (which is like YAML::Store but thread safe) is a *lot* slower than SQLite. To the point where you might not have to worry about rate limiting, because PStore access spreads the activity anyway.